sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -78,7 +78,7 @@ def body(self) -> bytes:
         """
         return self.internal_response.content
 
-    def stream_download(self, pipeline: Pipeline, **kwargs) -> Iterator[bytes]:
+    def stream_download(self, pipeline: Pipeline, **kwargs) -> Iterator[bytes]: # pylint: disable=docstring-keyword-should-match-keyword-only
```

Comment:
Should we just include the keyword in the signature?

Reply:
yup let me do that